app/javascript/components/patient/laboratory/Laboratory.js

```diff
@@ -1,29 +1,21 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Button } from 'react-bootstrap';
+import { Button, Dropdown } from 'react-bootstrap';
 import axios from 'axios';
-
 import LaboratoryModal from './LaboratoryModal';
 import reportError from '../../util/ReportError';
+import DeleteDialog from '../../util/DeleteDialog';
 
 class Laboratory extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      showModal: false,
+      showEditModal: false,
+      showDeleteModal: false,
       loading: false,
     };
   }
 
-  toggleModal = () => {
-    this.setState(state => {
-      return {
-        showModal: !state.showModal,
-        loading: false,
-      };
-    });
-  };
-
   submit = lab => {
```

Comment:
can you rename this `handleEditSubmit` or whatever makes sense based on my comment above?

Reply:
Changed to handleLabSubmit since it would be used for Add too